LandSandBoat/server
# Improve functionality of Amphiptere mobs

**_I affirm:_**

## What does this pull request do?

Aims to correct certain inconsistencies in various Amphiptere mobs.
Sets up basic functionality to NM: Zirnitra
Adds missing mob skill scripts
Sets the correct animationSub for this family in mob_pools so they don't spawn grounded (they should default to in flight). 

## Steps to test these changes
!zone Xarcabard_S
Run forward just a bit until you encouter an Amphiptere mob, it should be flying now
You aggro it and engage, !tp 3000 several times to see the new TP moves.

!spawnmob 17338585 (Zirnitra)
!gotoid 17338585
You should be able to aggro and fight it.
Added some of the basic mechanics to this fight, including the "reaving wind knockback aura".
Mob will also print the "disappointed" message when the target wipes, however, I left a small TODO on this.

**Sources**
Silknoz' fantastic captures, both at 75 and 99: 
https://youtu.be/P0FQF4Sr4x0
https://youtu.be/DPP_ofLX6Z8
https://www.youtube.com/watch?v=P-nMxDG7XoQ
http://ffxidb.com/zones/137/zirnitra
https://www.bg-wiki.com/ffxi/Zirnitra
https://ffxiclopedia.fandom.com/wiki/Zirnitra
